contrib/tvmop/utils.py

```diff
@@ -20,7 +20,16 @@
 
 AllTypes = ["float32", "float64", "float16", "uint8", "int8", "int32", "int64"]
 RealTypes = ["float32", "float64", "float16"]
-
+acc_type_resolver = {
```

Comment:
`acc` usually means accumulation for reduce-like ops. Seems here it's only for unary element-wise ops. Shall we rename to something like `unary_result_type` to echo `np.result_type`? Also, I think it would be better to implement this as a function, instead of using a dict.

Reply:
Yeah. A function is better, but this dict is useful, it should be kept.